diff --git a/node_modules/jstz_sdk/jstz_sdk.js b/node_modules/jstz_sdk/jstz_sdk.js
index 41aabd5..8415b2d 100644
--- a/node_modules/jstz_sdk/jstz_sdk.js
+++ b/node_modules/jstz_sdk/jstz_sdk.js
@@ -1,4 +1,12 @@
-import * as wasm from "./jstz_sdk_bg.wasm";
+// import * as wasm from "./jstz_sdk_bg.wasm";
 export * from "./jstz_sdk_bg.js";
 import { __wbg_set_wasm } from "./jstz_sdk_bg.js";
-__wbg_set_wasm(wasm);
+fetch('./jstz_sdk_bg.wasm')
+    .then(response => response.arrayBuffer())
+    .then(bytes => WebAssembly.instantiate(bytes, {}))
+    .then(results => {
+        const wasm = results.instance;
+        // Use the WebAssembly instance
+        __wbg_set_wasm(wasm);
+    })
+    .catch(console.error);
